Sketch/SketchController.m

```diff
@@ -27,7 +34,7 @@ - (void) applicationDidFinishLaunching: (NSNotification *)notif
 	// base located in the bottom left.
 	//[mainItem setFlipped: NO];
 	[mainItem setSize: NSMakeSize(500, 400)];
-	[mainItem setLayout: [ETBezierFreeLayout layout]];
+	[mainItem setLayout: [ETBezierFreeLayout layoutWithObjectGraphContext: [self objectGraphContext]]];
```

Comment:
Yes, that's right. For COObject subclasses, the designated initializer should always take an object graph context as its last argument. So classes like ETBezierHandle and ETBezierHandleGroup should have their designated initializer changed.